src/data/finset/lattice.lean

```diff
@@ -908,27 +908,21 @@ begin
   exact s.min'_lt_max' ha hb hab
 end
 
-lemma max'_eq_of_dual_min' {s : finset α} (hs : s.nonempty) :
-  max' s hs = of_dual (min' (image to_dual s) (nonempty.image hs to_dual)) :=
-begin
-  rw [of_dual, to_dual, equiv.coe_fn_mk, equiv.coe_fn_symm_mk, id.def],
-  simp_rw (@image_id (order_dual α) (s : finset (order_dual α))),
-  refl,
-end
+lemma of_dual_min' {s : finset (order_dual α)} (hs : s.nonempty) :
+  of_dual (min' s hs) = max' (s.image of_dual) (hs.image _) :=
+by { convert rfl, exact image_id }
 
-lemma min'_eq_of_dual_max' {s : finset α} (hs : s.nonempty) :
-  min' s hs = of_dual (max' (image to_dual s) (nonempty.image hs to_dual)) :=
-begin
-  rw [of_dual, to_dual, equiv.coe_fn_mk, equiv.coe_fn_symm_mk, id.def],
-  simp_rw (@image_id (order_dual α) (s : finset (order_dual α))),
-  refl,
-end
+lemma of_dual_max' {s : finset (order_dual α)} (hs : s.nonempty) :
+  of_dual (max' s hs) = min' (s.image of_dual) (hs.image _) :=
+by { convert rfl, exact image_id }
 
-@[simp] lemma of_dual_max_eq_min_of_dual {a b : α} :
-  of_dual (max a b) = min (of_dual a) (of_dual b) := rfl
+lemma to_dual_min' {s : finset α} (hs : s.nonempty) :
+  to_dual (min' s hs) = max' (s.image to_dual) (hs.image _) :=
+by { convert rfl, exact image_id }
 
-@[simp] lemma of_dual_min_eq_max_of_dual {a b : α} :
-  of_dual (min a b) = max (of_dual a) (of_dual b) := rfl
```

Comment:
Where did the lemmas about `min` and `max` go?

Reply:
Oh.. I think these lemmas were supposed to be about `finset.max` not `_root_.max`, and that wasn't communicated well in the PR comments when it was added.

Reply:
Added now. They do not look great because `finset.min` returns an `option`, but they exist.